docs/hazmat/primitives/asymmetric/dh.rst

```diff
@@ -62,3 +65,86 @@ Diffie-Hellman key exchange
         :type: int
 
         The generator value.
+
+
+Key interfaces
+~~~~~~~~~~~~~~
+
+.. class:: DHParameters
+
+    .. versionadded:: 0.9
```

Comment:
For DSA we support generation of private keys from pregenerated parameters. What makes DH different here?